nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -277,7 +277,7 @@ void BtActionServer<ActionT>::populateErrorCode(
 {
   int highest_priority_error_code = std::numeric_limits<int>::max();
   for (const auto & error_code_id : error_code_ids_) {
-    // TODO(jwallace42): Using try catch due to
+    // Note: Using try catch due to
```

Comment:
Why not just delete all this comment? I don't see that the context is super important